packages/web3js-plugin/src/plugin.ts

```diff
@@ -24,6 +24,46 @@ export class OptimismPlugin extends Web3PluginBase {
     | Contract<typeof gasPriceOracleABI>
     | undefined
 
+  /**
+   * Retrieves the basefee scalar
+   */
+  public async getBaseFeeScalar<
+    ReturnFormat extends DataFormat = typeof DEFAULT_RETURN_FORMAT
+  >(returnFormat?: ReturnFormat) {
+    return Web3.utils.format(
+      { format: 'uint' },
+      await this._getPriceOracleContractInstance().methods.baseFeeScalar().call(),
```

Comment:
This might be wrong casing but the typechecker will figure this out once it this is rebased on top
```suggestion
      await this._getPriceOracleContractInstance().methods.basefeeScalar().call(),
```